client/src/domain/interview.tsx

```diff
@@ -52,6 +52,43 @@ export enum InterviewStatus {
   Canceled,
 }
 
+export enum InterviewResult {
+  Yes,
+  No,
+  Draft,
+}
```

Comment:
IMO it's a bit better to make it more readable than just numbers
```suggestion
export enum InterviewResult {
  Yes = 'yes',
  No = 'no',
  Draft 'draft',
}
```